public/components/navbar.html

```diff
@@ -1,13 +1,12 @@
-<nav class="navbar navbar-expand border border-dark">
+<nav
+  class="navbar navbar-expand-lg bg-body-tertiary text-light"
+  style="background-color: #2a2f35 !important"
```

Comment:
let's move this to `css/index.css` and add that to every page since it's common across the site

Reply:
Ok